travis-ymls/castor.travis.yml

```diff
@@ -0,0 +1,16 @@
+# ----------------------------------------------------------------------------
+# Package             : castor
+# Source Repo         : https://github.com/castor-data-binding/castor
+# Travis Job Link     : https://travis-ci.com/github/sanjaymsh/castor/builds/216765641
+# Created travis.yml  : No
+# Maintainer          : Sanjay Mishra <snjkmr32@gmail.com>
+# Script License      : Apache License, Version 2 or later
+# ----------------------------------------------------------------------------
+language: java
+arch:
+  - amd64
+  - ppc64le
+jdk:
+  - openjdk9
```

Comment:
On amd64, currently it uses oraclejdk, should we not leave it as it is instead of changing it to openjdk ? For ppc64le, it may be better to use jdk 11

Reply:
@gururajrkatti , It failing for amd64 if i am keeping as it is with below errors: 

install-jdk.sh 2020-06-02
Expected feature release number in range of 9 to 17, but got: 8
The command "~/bin/install-jdk.sh --target "/home/travis/oraclejdk8" --workspace "/home/travis/.cache/install-jdk" --feature "8" --license "BCL"" failed and exited with 3 during .